tsconfig.json

```diff
@@ -70,6 +70,6 @@
     "skipLibCheck": true,                           /* Skip type checking of declaration files. */
     "forceConsistentCasingInFileNames": true        /* Disallow inconsistently-cased references to the same file. */
   },
-  "include": ["**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": ["./**/*.ts"],
+  "exclude": ["./node_modules/**/*", "./@scripts/**/*", "./@types/**/*"]
```

Comment:
I've also made some changes to the tsconfig. Personally I've added a tsconfig.build.json that extends this one and has a few more includes/excludes rules.

For rerference;
```
{
  "extends": "./tsconfig.json",
  "exclude": [
    "node_modules",
    "cypress",
    "coverage",
    "@types",
    "**/*.spec.ts",
    "**/*.spec.tsx",
  ],
}

```